src/global/translations/en.json

```diff
@@ -588,10 +588,16 @@
       "actionSheetTitle": "Select document to store"
     },
     "documentView": {
+      "submitdocument": "Submit Document",
```

Comment:
Please use lowerCamelCase for string keys.

Reply:
Done